config/src/test/scala/com/typesafe/config/impl/ConfigTest.scala

```diff
@@ -858,7 +858,7 @@ class ConfigTest extends TestUtils {
         if (home != null) {
             assertEquals(home, conf.getString("system.home"))
         } else {
-            assertEquals(nullValue, conf.getObject("system").get("home"))
+            assertEquals(null, conf.getObject("system").get("home"))
```

Comment:
The tests were failing on windows when home was null because the assertion didn't match the documented api.
The method returns a null so now the test asserts null not nullValue